environment.yml

```
@@ -1,9 +1,9 @@
-name: coffea-env
+name: clib-env
```

Can you undo this change?
```suggestion
name: coffea-env
```